### PR TITLE
Add in the link for Auto Attendant diagnostic

### PR DESCRIPTION
**Pull request by dj1298**
Added in details and direct link to the Auto Attendant diagnostic that is available to the Admins.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-auto-attendant.md [hunk 1]
```diff
@@ -253,6 +253,37 @@ You can also use PowerShell to create and set up auto attendants. Here are the c
 - [Import-CsAutoAttendantHolidays](/powershell/module/skype/import-csautoattendantholidays)
 - [New-CsAutoAttendantCallableEntity](/powershell/module/skype/New-CsAutoAttendantCallableEntity)
 
+
+## More information
+
+If you have followed these steps, but have issues with the Auto Attendant, Microsoft has developed Teams-specific diagnostic scenarios that may help.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:259]:
```suggestion
If you have followed these steps, but have issues with the auto attendant, we have developed Teams-specific diagnostic scenarios that may help. 
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 2]
```diff
@@ -253,6 +253,37 @@ You can also use PowerShell to create and set up auto attendants. Here are the c
...
+### Non-administrators
+
+Please contact your [Microsoft 365 Administrator](/microsoft-365/admin/add-users/about-admin-roles?view=o365-worldwide&preserve-view=true). 
+
+### Administrators
+
+> [!NOTE]
+> This feature is not available for Microsoft 365 Government, Microsoft 365 operated by 21Vianet, or Microsoft 365 Germany.
+
+While you’re logged in as an administrator, visit your Microsoft 365 admin center. In the navigation pane, select Show all > Support > New service request. After you briefly describe your issue (for example, “Teams Auto Attendant Troubleshooting”), the system determines whether a diagnostic scenario matches your issue.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:270]:
```suggestion
While you’re logged in as an administrator, navigate to the  Microsoft 365 admin center. In the navigation pane, select **Show all** > **Support** > **New service request**. After you have briefly described your issue (for example, “Teams auto attendant troubleshooting”), the system determines whether a diagnostic scenario matches your issue.
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 3]
```diff
@@ -253,6 +253,37 @@ You can also use PowerShell to create and set up auto attendants. Here are the c
...
+Or you can click this link to get to the diagnostic directly!
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:271]:
```suggestion
Or you can select this link to get to the diagnostic directly.
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 4]
```diff
@@ -253,6 +253,37 @@ You can also use PowerShell to create and set up auto attendants. Here are the c
...
+
+1. Select **Run Tests** below, which will populate the diagnostic in the Microsoft 365 Admin Center.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:273]:
```suggestion
1. Select **Run Tests**, which will populate the diagnostic in the Microsoft 365 admin center. 
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 5]
```diff
@@ -253,6 +253,37 @@ You can also use PowerShell to create and set up auto attendants. Here are the c
...
+> This feature is not available for Microsoft 365 Government, Microsoft 365 operated by 21Vianet, or Microsoft 365 Germany.
+
+While you’re logged in as an administrator, visit your Microsoft 365 admin center. In the navigation pane, select Show all > Support > New service request. After you briefly describe your issue (for example, “Teams Auto Attendant Troubleshooting”), the system determines whether a diagnostic scenario matches your issue.
+Or you can click this link to get to the diagnostic directly!
+
+1. Select **Run Tests** below, which will populate the diagnostic in the Microsoft 365 Admin Center. 
+
+    > [Run Tests: Teams Auto Attendant](https://aka.ms/TeamsAADiag)
+
+2. In the diagnostic form, enter your Auto Attendant Resource email address, and then select **Run Tests**.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:277]:
```suggestion
2. In the diagnostic form, enter your auto attendant resource email address, and then select **Run Tests**.
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 6]
```diff
@@ -253,6 +253,37 @@ You can also use PowerShell to create and set up auto attendants. Here are the c
...
+![image](https://user-images.githubusercontent.com/11858030/133628641-c5277c02-89f8-4599-9c3c-1cf9a064c73f.png)
+
+
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:280]:
```suggestion
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 7]
```diff
@@ -253,6 +253,37 @@ You can also use PowerShell to create and set up auto attendants. Here are the c
...
+3. If the test detects configuration issues, it will display links to help resolve those issues.
+
+![image](https://user-images.githubusercontent.com/11858030/133628705-e04d84f9-a468-4189-b3a8-9ac11a8b6bec.png)
+
+
+
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:286]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dj1298, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@dj1298 opening support tickets is covered [elsewhere on Docs](https://docs.microsoft.com/en-us/microsoft-365/business-video/get-help-support?view=o365-worldwide). We wouldn't include details on that process flow here.

**Pull request closed without merging**